app/editor/src/components/form/text/Text.tsx

```diff
@@ -30,6 +30,12 @@ export interface ITextProps extends InputHTMLAttributes<HTMLInputElement> {
    * Form field size.
    */
   width?: FieldSize;
+  /**
+   * Function to format the value.
+   */
+  formatter?: (
```

Comment:
Now we can format the value.  I don't think this will handle masks though.  It's just a quick and dirty way to format values after a render.